hi-ml-histopathology/Makefile

```diff
@@ -89,7 +89,6 @@ pytest_fast: pip_test call_pytest_fast
 # run pytest with coverage on package, and format coverage output as a text file, assuming test requirements already installed
 call_pytest_and_coverage:
 	pytest --cov=histopathology  --cov-branch --cov-report=html --cov-report=xml --cov-report=term-missing --cov-config=.coveragerc testhisto
-	pytest --cov=SSL  --cov-branch --cov-report=html --cov-report=xml --cov-report=term-missing --cov-config=.coveragerc testSSL
```

Comment:
Not sure why this was removed? Should this change be a part of the PR?

Reply:
It was overwriting the  coverage.txt file so we weren't seeing the coverage report. This is a temporary solution, to check the histo coverage

Reply:
it's in the PR description, hope it's clear enough, the github issue is left open. The plan is to move SSL outside of pathology, this is why we thought temporary fix was good enough